test.js

```diff
@@ -13,7 +13,7 @@ function MyPersistence () {
 
   // copied from Memory
   this._retained = []
-  this._subscriptions = []
+  this._subscriptions = new Map()
```

Comment:
This is used for the tests only and is copied over from Memory persistence in aedes-persistence. A bunch of methods are copied over from Memory persistence too, including subscriptionsByClient which needs all the subscriptions, including qos 0 ones which aren't stored in the QlobberSub.